### ISSUE TITLE
The dropdown list "group""Assigned to" drop-down list is empty

### ISSUE THREAD

**Issue by Riesuu** [pluginsGLPI/escalade#98]
GLPI 9.5.4

The "Assigned to" drop-down list is empty, on the other hand, no problem with observers.

![2021-03-10 09_26_18-Clipboard](https://user-images.githubusercontent.com/17006344/110599039-c4a7e080-8182-11eb-8cf5-79010f1f3d4a.png)

php-errors:

```
`[2021-03-10 06:26:04] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 07:55:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 08:16:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 08:21:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 08:23:01] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"2256@sai-glpi-prd-01"} 
[2021-03-10 08:24:22] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"2256@sai-glpi-prd-01"} 
[2021-03-10 08:28:30] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"2203@sai-glpi-prd-01"} 
[2021-03-10 08:28:59] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"2203@sai-glpi-prd-01"} 
[2021-03-10 08:29:31] glpiphplog.DEBUG: Html::compileScss() in /var/www/html/glpi/inc/html.class.php line 7495
css/palettes/icecream.scss is new, loading  {"user":"2051@sai-glpi-prd-01"} 
[2021-03-10 08:29:31] glpiphplog.DEBUG: Html::compileScss() in /var/www/html/glpi/inc/html.class.php line 7495
css/styles.scss is new, loading  {"user":"2051@sai-glpi-prd-01"} 
[2021-03-10 08:29:34] glpiphplog.DEBUG: Html::compileScss() in /var/www/html/glpi/inc/html.class.php line 7495
css/dashboard.scss is new, loading  {"user":"2051@sai-glpi-prd-01"} 
[2021-03-10 08:32:38] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"2256@sai-glpi-prd-01"} 
[2021-03-10 08:40:09] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"2203@sai-glpi-prd-01"} 
[2021-03-10 08:42:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 08:45:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 08:49:52] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"2203@sai-glpi-prd-01"} 
[2021-03-10 08:57:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 09:02:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 09:03:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 09:05:02] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"3929@sai-glpi-prd-01"} 
[2021-03-10 09:10:42] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"2256@sai-glpi-prd-01"} 
[2021-03-10 09:17:03] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactiveprofile" in /var/www/html/glpi/inc/commonitilobject.class.php at line 2708
  Backtrace :
  inc/itilfollowup.class.php:266                     CommonITILObject::isAllowedStatus()
  inc/commondbtm.class.php:1151                      ITILFollowup->post_addItem()
  inc/mailcollector.class.php:877                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 09:17:03] glpiphplog.CRITICAL:   *** Uncaught Exception TypeError: array_key_exists(): Argument #2 ($array) must be of type array, null given in /var/www/html/glpi/inc/commonitilobject.class.php at line 2708
  Backtrace :
  inc/itilfollowup.class.php:266                     CommonITILObject::isAllowedStatus()
  inc/commondbtm.class.php:1151                      ITILFollowup->post_addItem()
  inc/mailcollector.class.php:877                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 09:29:42] glpiphplog.ERROR: CommonITILObject::getTemplateFieldName() in /var/www/html/glpi/inc/commonitilobject.class.php line 7950
Missing type for Ticket template!  {"user":"2045@sai-glpi-prd-01"} 
[2021-03-10 09:30:53] glpiphplog.WARNING: Test logger  
[2021-03-10 09:31:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 09:33:02] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpiactive_entity" in /var/www/html/glpi/marketplace/formcreator/inc/issue.class.php at line 400
  Backtrace :
  inc/commondbtm.class.php:3644                      PluginFormcreatorIssue->rawSearchOptions()
  inc/commondbtm.class.php:3951                      CommonDBTM->searchOptions()
  inc/commondbtm.class.php:4023                      CommonDBTM->getSearchOptionByField()
  inc/commondbtm.class.php:1121                      CommonDBTM->filterValues()
  marketplace/formcreator/hook.php:350               CommonDBTM->add()
  inc/plugin.class.php:1350                          plugin_formcreator_hook_add_ticket()
  inc/commondbtm.class.php:1185                      Plugin::doHook()
  inc/mailcollector.class.php:824                    CommonDBTM->add()
  inc/mailcollector.class.php:1746                   MailCollector->collect()
  inc/crontask.class.php:845                         MailCollector::cronMailgate()
  front/cron.php:83                                  CronTask::launch()
  
[2021-03-10 09:41:14] glpiphplog.DEBUG: Html::compileScss() in /var/www/html/glpi/inc/html.class.php line 7495
css/styles.scss is new, loading  {"user":"2051@sai-glpi-prd-01"} 
[2021-0`
```

I just deactivated the escalation plugin and it worked. When installing the update, I get an error.

```
SQL Error "1146": Table 'glpidb.glpi_plugin_useditemsexport_configs' doesn't exist in query "SELECT * FROM `glpi_plugin_useditemsexport_configs` WHERE `glpi_plugin_useditemsexport_configs`.`id` = '1' LIMIT 1"
PHP Warning (2): Undefined array key "is_active" in /var/www/html/glpi/marketplace/useditemsexport/setup.php at line 65
```

I just deactivated the escalation plugin and it worked. When installing the update, I get an error.

```
SQL Error "1146": Table 'glpidb.glpi_plugin_useditemsexport_configs' doesn't exist in query "SELECT * FROM `glpi_plugin_useditemsexport_configs` WHERE `glpi_plugin_useditemsexport_configs`.`id` = '1' LIMIT 1"
PHP Warning (2): Undefined array key "is_active" in /var/www/html/glpi/marketplace/useditemsexport/setup.php at line 65
```

Then I found the same plugins in /marketplace and /plugins. Probably because I installed it manually and wanted to update via the marketplace.

So I removed the escalator/formcreator plugin from the marketplace. To reinstall it manually. Then, I had to clean the plugins via configuration>plugins to have no more error.

Currently, the problem is still present, but I don't have any more error in debug mode...

**Comment by Riesuu** [comment]
https://github.com/glpi-project/glpi/issues/8806

**Comment by cedric-anne** [comment]
Which version of the plugin do you have ? Did you tried with version 2.6.1 ?

**Comment by Riesuu** [comment]
Yes, now i have 2.6.1 and problem is present.

**Comment by Riesuu** [comment]
Hi,
problem is resolved since update glpi 9.5.5
